mac-setup/complete-build.sh

```diff
@@ -1,20 +1,25 @@
-#!/bin/bash -x
-export PATH="$HOME/.new_local/bin:$PATH"
+#!/usr/bin/env bash
 
-# Build for macOS, without GTK and without Poppler
+set -e
+set -o pipefail
+
+# Build and package the application
 
 if [ $# -eq 0 ]; then
-  echo 'Please provide the path of your gtk installation'
+  echo 'Please provide the path of your gtk prefix parent (example: ~/gtk)'
   exit 1
 fi
 
-export PATH="$HOME/.local/bin:$1/inst/bin:$PATH"
+root_dir=$(readlink -f "$(dirname "$0")"/..)
+prefix_parent="$1"
+[ ! -d "$prefix_parent" ] && exit "$prefix_parent doesn't exist."
```

Comment:
This just caught my eye, but Bash only allows the single argument of `exit` to be an integer representing the exit code, not a string

Reply:
Thanks for the catch. I've been writing too much python lately, where `sys.exit("message")` does exactly this...